### DISCUSSION TITLE
posts/2024/08/getting-one-august/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#54] [Announcements]
# posts/2024/08/getting-one-august/

Why I decided to issue this short one.

https://www.brycewray.com/posts/2024/08/getting-one-august/?utm_campaign=RSS&utm_source=RSS&utm_medium=RSS

**Comment by bobmonsour** [comment]
Glad to hear that your recovery is going well!

**Reply by brycewray** [reply]
@bobmonsour Thanks!

**Comment by hryggrbyr** [comment]
Seconding Bob; glad recovery is going well 😊

**Reply by brycewray** [reply]
@hryggrbyr Thanks!

**Comment by ElastiCourse** [comment]
I'm glad you are recovering and writing again Bruce. I can't wait to read more of your inspiring articles on Hugo and other topics. Stay safe!

**Reply by brycewray** [reply]
@ElastiCourse Thanks!
